### DISCUSSION TITLE
有没有人开发了wordpress插件

### DISCUSSION THREAD

**Discussion by Alicexs** [lsky-org/lsky-pro#390] [想法]
有没有大佬开发了wordpress插件对接兰空图床，网上只看到了Typecho 插件兰空图床图片外链，能够将 Typecho博客系统图片单独托管于兰空图床。

**Comment by a523042524** [comment]
https://www.qqworld.org/?promotion=1195
这个可以

**Reply by Alicexs** [reply]
这个插件功能太多，感觉太臃肿了，感觉可以开发一个wordpress插件做一个图片外链
